plugins/spigot/src/main/java/com/managemc/spigot/command/util/CommandNodeMap.java

```diff
@@ -126,6 +126,19 @@ public CommandNodeMap(SpigotPluginConfig config) {
               }}
           )
           .build());
+
+      add(CommandNode.builder()
+          .keyword("reports")
+          .action(null)
+          .children(
+              new HashSet<CommandNode>() {{
+                add(CommandNode.builder().keyword("create")
+                    .handler(new CmdReportsCreate(config))
+                    .action(null)
```

Comment:
The permission will go both here and above (line 132) for commands that require permission. We run permission checks before calling the API so that the player gets immediate feedback, but all permissions are also validated server-side.